tallycash/extension
# Add support for typed data signing on Ledger accounts

Resolves #1032

# Summary

Naive happy path implemented when typed data signing is required with ledger.

# Tests

- Check non-Ledger account signing
- Check Ledger account signing
